### DISCUSSION TITLE
How to creat a oblique cylinder?

### DISCUSSION THREAD

**Discussion by Pandakii** [AMReX-Combustion/PeleC#824] [Q&A]
Hi everyone,
The implicit function `CylinderIF` can be used to construct a cylinder that is parallel to the coordinate axis. How to make use of implicit functions to creat a oblique cylinder with a angle relative to the coordinate axis? Or, is it possible to rotate the normal cylinder around a point?

**Comment by marchdf** [comment]
We have a rotated cylinder option in PeleC: https://github.com/AMReX-Combustion/PeleC/blob/development/Source/Geometry.cpp#L293.  `eb2.geom_type = "rotated_cylinder"` and you can see the options it is looking for in that source code I linked to. E.g.: `eb2.cylinder_radius = 1.0` and `eb2.cylinder_rotation_axe = 2` can be used in your input file.
